online-store/src/components/sort/sort.ts

```diff
@@ -0,0 +1,63 @@
+import { productsArrayI } from '../types';
+import { productsData, CreateProductCard } from '../cards/cards';
+import { resultData } from '../filter/filter';
+import { searchResult } from '../search/search';
+import { searchParams } from '../routing/routing';
+
+export function SortProductCard(trigger: string, arraySort?: productsArrayI[]) {
+  // 'now' для выпослнения сразу после фильтрации
+
+  const sort = document.querySelector('.sort-input') as HTMLInputElement;
+  let sortArrproducts: Array<productsArrayI> = [];
+
+  function update() {
```

Comment:
it is important to use meaningful names for functions. Such common names make the code difficult to read. In addition, you should always specify the return data type